src/prototype_room_my.js

```diff
@@ -229,7 +228,7 @@ Room.prototype.checkCanHelp = function() {
   }
 
   let nearestRoom = this.memory.nearestRoom;
-  if (!nearestRoom || !Memory.rooms[nearestRoom].needHelp) {
+  if (!nearestRoom || !Memory.rooms[nearestRoom] || !Memory.rooms[nearestRoom].needHelp) {
```

Comment:
:+1: I think you have to include `Fixes #164' in the commit message, so that the issue get closed.